presto-hive/src/main/java/io/prestosql/plugin/hive/azure/HiveAzureConfig.java

```diff
@@ -36,6 +39,11 @@ public void validate()
                     !isNullOrEmpty(wasbAccessKey) && !isNullOrEmpty(wasbStorageAccount),
                     "If one of wasbAccessKey, wasbStorageAccount is set, both must be set");
         }
+        if (adlClientId != null || adlCredential != null || adlRefreshUrl != null) {
+            checkState(
+                    !isNullOrEmpty(adlClientId) && !isNullOrEmpty(adlCredential) && !isNullOrEmpty(adlRefreshUrl),
+                    "If one of adlClientId, adlCredential, adlRefreshUrl, all must be set for adlAccessTokenProviderType=CLIENT_CREDENTIAL");
```

Comment:
the message refers to `adlAccessTokenProviderType` setting which seems not to exist

at the point of this commit, the `AdlAccessTokenProviderType` has just one option and is unused, so can be removed